src/core/layout/qgslayoutitemattributetable.cpp

```diff
@@ -941,12 +942,7 @@ bool QgsLayoutItemAttributeTable::readPropertiesFromElement( const QDomElement &
 
   //map
   mMapUuid = itemElem.attribute( QStringLiteral( "mapUuid" ) );
-  if ( mMap )
-  {
-    disconnect( mMap, &QgsLayoutItemMap::extentChanged, this, &QgsLayoutTable::refreshAttributes );
-    disconnect( mMap, &QgsLayoutItemMap::mapRotationChanged, this, &QgsLayoutTable::refreshAttributes );
-    mMap = nullptr;
```

Comment:
Is the issue here a dangling pointer to mMap? If so, making mMap a QPointer might be a simpler/safer fix

Reply:
Good point.